src/assets/js/site.js

```diff
@@ -523,6 +551,121 @@ var reCaptcha3ApiLoad = function() {
     }
 }
 
+var turnstileApiLoad = () => {
+    var a = document.createElement("script");
+    a.type = "text/javascript";
+    a.defer = true;
+    "undefined" !== typeof site_nonce && a.setAttribute('nonce', site_nonce);
+    a.src = "//challenges.cloudflare.com/turnstile/v0/api.js?onload=turnstileOnLoad&render=explicit";
```

Comment:
@danlt150 để chạy được cái này cần phải thêm `challenges.cloudflare.com` hoặc `*.cloudflare.com` vào mục script-src trong CSP khi cài đặt.

Ngoài ra turnstile này anh thấy yêu cầu thêm quyền picture-in-picture nên trong cấu hình PP khi cài đặt cũng cần bổ sung thêm `https://challenges.cloudflare.com` cho nó

Reply:
Khi em cài đặt ở chế độ DEV thì cái CSP bị tắt, cái này em vào bật lại sẽ thấy ngoài site không hoạt động được